scripts/entrypoint.dev.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
 python3 manage.py migrate
-python3 manage.py runserver 0.0.0.0:8000
-python3 manage.py shell < data_management.py
+#python3 manage.py runserver 0.0.0.0:8000
+python manage.py runserver_plus --cert-file /cert/amcr-test-local.crt 0.0.0.0:8000
```

Comment:
Tady by asi mělo být `python3`

Reply:
@jnihnat mám čekat na úpravu, nebo je to ok?

Reply:
upraveno